renderer/viewer/lib/mesher/models.ts

```
@@ -387,7 +387,12 @@ function renderElement (world: World, cursor: Vec3, element: BlockElement, doAO:
     const aos: number[] = []
     const neighborPos = position.plus(new Vec3(...dir))
     // 10%
-    const baseLight = world.getLight(neighborPos, undefined, undefined, block.name) / 15
+    const { smoothLighting, shadingTheme } = world.config
+    const faceLight = world.getLight(neighborPos, undefined, undefined, block.name)
+    const sideShading = (shadingTheme === 'high-contrast') ?
+    (0.8 + 0.5*Math.max(0.0,0.66*dir[0]+0.66*dir[1]+0.33*dir[2])) : //old directional light behavior
```

ESLint

[error] Expected indentation of 6 spaces but found 4.

(`@stylistic/indent`)

---

[error] Operator '*' must be spaced.

(`@stylistic/space-infix-ops`)

---

[error] Don't use a zero fraction in the number.

(unicorn/no-zero-fractions)